src/docs/product/dashboards/index.mdx

```diff
@@ -8,7 +8,7 @@ redirect_from:
 description: "Sentry's Dashboards provide a single view of your application's health, including the ability to navigate issues and performance across multiple projects."
 ---
 
-Dashboards provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).
+<SandboxLink scenario="dashboards" projectSlug="react">Dashboards</SandboxLink> provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).
```

Comment:
@Neo-Zhixing do you have a solution for this one? This is another case where the HTML tags in the Sandbox link are breaking the link for the markdown. Ideally, we don't use html links between the docs themselves, rather just these short links.

Reply:
I think gatsby-mdx is interpreting `<SandboxLink>` as a block-level component making the links render incorrectly.
There are two workarounds:
- Don't use `<SandboxLink>` as the first thing in a paragraph. Put something else before it.
- Use `<a>` tags for links.

Obviously this is not ideal, so if it becomes a huge problem in the future, let me know and I'll go into the upstream packages to fix it.

Reply:
I've thrown a word in front of the link so hopefully that will fix this.

Reply:
Also @Neo-Zhixing, I'd venture to say this behaviour is already a problem (is a real problem in lists of links). Can I log it as an issue and assign it to you?

Reply:
Sure, put it on growth backlog and we will revisit this in the future.